setup/docker/build.sh

```diff
@@ -13,12 +13,13 @@ tmp="./setup/docker/tmp"
 
 mkdir -p ${tmp}
 
-cat ./setup/unix/traccar.xml | awk '/web.path/ && !modif { printf("    <entry key=\"web.debug\">true</entry>\n"); modif=1 } {print}' > ${tmp}/traccar.xml
+cat ./setup/traccar.xml | awk '/web.path/ && !modif { printf("    <entry key=\"web.debug\">true</entry>\n"); modif=1 } {print}' > ${tmp}/traccar.xml
```

Comment:
Makes sense. In that case, I suggest changing this line to

`cat ./setup/traccar.xml | awk '/config.default/ && !modif { print;printf("    <entry key=\"web.debug\">true</entry>\n");next; modif=1 } {print}'  > ${tmp}/traccar.xml`

essentially adding the `web.debug` line after the `config.default` line